.github/workflows/deploy_latest_release.yml

```diff
@@ -0,0 +1,59 @@
+name: SalesforceCommerceCloud/pwa-kit/deploy_latest_release
+on:
+  # For testing - to be removed
+  push:
+    branches:
+      - 'mrt-test-ci-deploy'
+  # Run this workflow when a new Github release is published
+  release:
+    types: [released]
+
+jobs:
+  check-latest-release:
+    runs-on: ubuntu-latest
+    steps:  
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Check if latest release
+        uses: ./.github/actions/check_if_latest_release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  deploy-bug-bounty:
+    needs: check-latest-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup node
+        id: setup_node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: "npm"
+
+      - name: Install Monorepo Dependencies
+        run: |-
+          # Install node dependencies
+          node ./scripts/gtime.js monorepo_install npm ci
+
+      - name: Generate Retail App Demo
+        uses: ./.github/actions/e2e_generate_app
+        with:
+          PROJECT_KEY: "retail-react-app-bug-bounty"
+
+      - name: Create MRT credentials file
+        uses: "./.github/actions/create_mrt"
+        with:
+          mobify_user: ${{ secrets.MOBIFY_STG_CLIENT_USER }}
+          mobify_api_key: ${{ secrets.MOBIFY_STG_CLIENT_API_KEY }}
+
+      - name: Push Bundle to MRT (Bug Bounty)
+        uses: "./.github/actions/push_to_mrt"
+        with:
+          CWD: "../generated-projects/retail-react-app-bug-bounty"
+          TARGET: testing
+          PROJECT: pwa-kit
+          FLAGS: --cloud-origin https://cloud-testing.mobify-staging.com -c ~/.mobify --wait
```

Comment:
Would it be useful if we add the tag name to the bundle name so that we know which code version the bundle is using if any bug is found?

Reply:
Added!